docs/en_US/Blog/HPOComparison.md

```diff
@@ -0,0 +1,323 @@
+# Hyperparameter Optimization Comparison
+*Posted by Anonymous Author*
+
+Comparison of Hyperparameter Optimization algorithms on several problems.
+
+Hyperparameter Optimization algorithms are list below:
+
+- Random Search
+- Grid Search
+- Evolution
+- Anneal
+- Metis
+- TPE
+- SMAC
+- HyperBand
+- BOHB
+
+All algorithms run in NNI local environment.
+
+## First Fake Function
```

Comment:
First Fake Function? How about “Optimize convex function”?

Reply:
if we remove this part, we don't have to care about the title...

Reply:
I change the title of this section and still keep it.